orion-server/src/main/java/com/pinterest/orion/core/kafka/BrokersetState.java

```diff
@@ -0,0 +1,53 @@
+package com.pinterest.orion.core.kafka;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class BrokersetState {
+    private String brokersetAlias;
+    private List<List<Integer>> brokersetRanges = new ArrayList<>();
+    private List<String> brokerIds;
```

Comment:
Are we keeping redundant information in these two variables? Isn't `brokerIds` union of all `brokersetRanges`?

Reply:
The ranges are from the configuration. The ids are from the cluster info.

There can be a condition that brokerRange contains a broker that does not exist. Now, that means the brokerset is valid. Once we have more condition check, the broker can be no show for other reasons. 

Will add comments for these two variables along with other changes.